src/workloads/c2c/MongosyncScripts.yml

```diff
@@ -0,0 +1,95 @@
+SchemaVersion: 2018-07-01
+
+# This is a collection of helper scripts used to communicate with mongosync. To use them,
+# use LoadConfig and load the script you would like to use in your actor's phase.
+# For example:
+# - Name: StartMongosync
+#   Type: ExternalScriptRunner
+#   Threads: 1
+#   Phases:
+#     - LoadConfig:
+#       Path: ./MongosyncScripts.yml
+#       Key: StartMongosync
+#
+# Note that the polling scripts start with set -eou pipefail so that any
+# subcommand run during the script failing will cause the entire script to fail.
+# This way if mongosync crashes during the test, it will stop the test execution
+# immediately and genny will bubble up the error and fail the test
+#
+# For curl commands we pass -s and -S so that an errors are still reported (-S)
+# but download progress stats are not logged to stderr
+#
+StartMongosync:
+  Repeat: 1
+  Command: sh
+  MetricsName: StartMongosync
+  Script: |
+    curl -s -S -X POST http://10.2.0.160:27182/api/v1/start \
+      -H 'Content-Type: application/json' \
+      -d '{"Source": "cluster0", "Destination": "cluster1"}'
+
+PollForCEA:
+  Repeat: 1
+  Command: sh
+  MetricsName: PollForCEA
+  Script: |
+    set -eou pipefail
+
+    getState() {
+      curl -s -S http://10.2.0.160:27182/api/v1/progress | jq '.progress.info' --raw-output
```

Comment:
I think it would be neat if we added a way to pass in environment variables in the future, so we can pass down this data from DSI. I wanted to keep this simple for now though, as this might change a lot during the project

Reply:
Oh cool I was just thinking about the same thing above.